qutebrowser/browser/webengine/spell.py

```diff
@@ -21,26 +21,46 @@
 
 import glob
 import os
+import re
 
+from qutebrowser.utils import log
 from PyQt5.QtCore import QLibraryInfo
 
 
+def version(filename):
+    """Extract the version number from the dictionary file name."""
+    version_re = re.compile(r"""
+        .+-(?P<version>[0-9]+-[0-9]+?)\.bdic
+    """, re.VERBOSE)
```

Comment:
Seems fine to put this in one line: `version_re = re.compile(r".+-(?P<version>[0-9]+-[0-9]+?)\.bdic")`